packages/client/internals/NavControls.vue

```diff
@@ -118,7 +120,7 @@ if (__SLIDEV_FEATURE_DRAWINGS__)
           <carbon:user-speaker />
         </RouterLink>
 
-        <button v-if="__DEV__ && !isPresenter" class="slidev-icon-btn <md:hidden" @click="showEditor = !showEditor">
+        <button v-if="__DEV__ && !isPresenter && !passwordSet && configs.codeEditor" class="slidev-icon-btn <md:hidden" @click="showEditor = !showEditor">
```

Comment:
Instead of only checking if password is set or not, maybe we can check if password is correct ?